modules/040-control-plane-manager/docs/FAQ_RU.md

```diff
@@ -4,7 +4,58 @@ title: "Управление control plane: FAQ"
 
 ## Как добавить master-узел?
 
-Поставить на узле кластер label `node-role.kubernetes.io/master: ""`, все остальное произойдет полностью автоматически.
+### Cтатичный или гибридный кластер
+Чтобы добавить статичный master-узел в кластер, выполните следующие действия:
+- [Добавьте](https://deckhouse.io/ru/documentation/v1/modules/040-node-manager/faq.html#как-автоматически-добавить-статичный-узел-в-кластер) в кластер статичный узел. `NodeGoup` может быть любая, с параметром `NodeType` — `Static`.
+- Поставьте на добавленном узле кластера label `node-role.kubernetes.io/master: ""`.
+
+Все остальные действия будут выполнены автоматически, дождитесь их завершения — появления master-узла в статусе Ready.  
+
+### Облачный кластер
+
+> Перед добавлением узлов, убедитесь в наличии необходимых квот!
+
+Чтобы добавить в облачный кластер один или несколько master-узлов, выполните следующие действия:
+- Определите версию и редакцию Deckhouse используемую в кластере. Для этого, выполните на master-узле, или на хосте с настроенным доступом kubectl в кластер:
+  ```shell
+  kubectl -n d8-system get deployment deckhouse \
+  -o jsonpath='versio-{.metadata.annotations.core\.deckhouse\.io\/version}, revision-{.metadata.annotations.core\.deckhouse\.io\/edition}' \
+  | tr '[:upper:]' '[:lower:]'
+  ```
+    
+  Пример вывода команды:
```

Comment:
This block is excessive. You can move the example to the upper block and delete this one.

Reply:
Done